controllers/uninstall.go

```diff
@@ -51,6 +51,14 @@ var (
 				types.NamespacedName{Name: "multiclusterhub-repo", Namespace: m.Namespace},
 				schema.GroupVersionKind{Group: "", Kind: "Service", Version: "v1"},
 			),
+			newUnstructured(
+				types.NamespacedName{Name: "searchcustomizations.search.open-cluster-management.io"},
+				schema.GroupVersionKind{Group: "search.open-cluster-management.io", Kind: "CustomResourceDefinition", Version: "v1alpha1"},
+			),
+			newUnstructured(
+				types.NamespacedName{Name: "searchoperators.search.open-cluster-management.io"},
+				schema.GroupVersionKind{Group: "search.open-cluster-management.io", Kind: "CustomResourceDefinition", Version: "v1alpha1"},
+			),
```

Comment:
```suggestion
			newUnstructured(
				types.NamespacedName{Name: "searchcustomizations.search.open-cluster-management.io"},
				schema.GroupVersionKind{Group: "apiextensions.k8s.io", Kind: "CustomResourceDefinition", Version: "v1"},
			),
			newUnstructured(
				types.NamespacedName{Name: "searchoperators.search.open-cluster-management.io"},
				schema.GroupVersionKind{Group: "apiextensions.k8s.io", Kind: "CustomResourceDefinition", Version: "v1"},
			),
```

Reply:
@JakobGray apologies I did change to this and forgot to add to my commit, thanks for the catch!